go.mod

```diff
@@ -7,6 +7,7 @@ replace github.com/VictoriaMetrics/fastcache => ./fastcache
 replace github.com/ethereum/go-ethereum => ./go-ethereum
 
 require (
+	cloud.google.com/go/storage v1.43.0
```

Comment:
Are we okay adding this as a dependency? cc @PlasmaPower @tsahee

Reply:
Yeah it looks like it's an official Google Cloud library so it sounds good to me